mods/lord/Entities/npc/e_deputy_mob.lua

```diff
@@ -0,0 +1,285 @@
+local S = minetest.get_translator("npc")
+local esc = minetest.formspec_escape
+
+---show mob answers
+---@param clicker Player player
+---@param item table current item we display
+local function show_answer(clicker, item)
+	--- 
+	local player = clicker:get_player_name()
+	local formspec = "size[8,5]"
+	formspec = formspec.."textarea[0.5,0;7.5,4;;;"..esc(item.answer).."]"
+	formspec = formspec.."button[0.25,4;7.5,1;return_to_main;"..esc(S("Back")).."]"
+	minetest.show_formspec(player, "npc:static_guide_answer", formspec)
+end
+
+---edit mob answers form
+---@param clicker Player player
+---@param item table current item which we edit
+local function edit_answer(clicker, item)
+	--- 
+	local player = clicker:get_player_name()
+	local formspec = "size[8,9]"
+	formspec = formspec.."field[0.5,0.0;0,0;old_label;;"..item["label"].."]"
+	formspec = formspec.."field[0.5,0.5;7.5,1;edit_label;;"..item["label"].."]"
+	formspec = formspec.."field[0.5,1.5;7.5,1;edit_question;;"..esc(item["question"]).."]"
+	formspec = formspec.."textarea[0.5,2.5;7.5,2.75;edit_answer;;"..esc(item["answer"]).."]"
+	formspec = formspec.."button[0.25,5;7.5,1;return_to_main;"..esc(S("Back")).."]"
+	formspec = formspec.."button[0.25,6;7.5,1;save_question;"..esc(S("Save")).."]"
+	formspec = formspec.."button[0.25,7;7.5,1;delete_question;"..esc(S("Delete")).."]"
+	if item["enabled"] then
+		formspec = formspec.."button[0.25,8;7.5,1;hide_question;"..esc(S("Hide")).."]"
+	else
+		formspec = formspec.."button[0.25,8;7.5,1;show_question;"..esc(S("Show")).."]"
+	end
+	minetest.show_formspec(player, "npc:edit_guide_answer", formspec)
+end
+
+---form content when display dialog
+---@param self table mob object
+---@param width number content wifth
+---@param pos number current position
+---@return string, number
+local function user_mob_content(self, width, pos)
+	--- 
+	local formspec = ""
+	local bw = width - 0.5
+	for _, item in ipairs(self.questions) do
+		if item["enabled"] then
+			formspec = formspec.."button[0.25,"..pos..";"..bw..",1;"..item["label"]..";"..item["question"].."]"
+			pos = pos + 1
+		end
+	end
+	return formspec, pos
+end
+
+---form content when edit mob
+---@param self table mob object
+---@param width number width of the content
+---@param pos number current position
+---@return string, number
+local function admin_mob_content(self, width, pos)
+	--- 
+	local bw = width - 0.5
+	local formspec = ""
+	for _, item in ipairs(self.questions) do
+		formspec = formspec.."button[0.25,"..pos..";"..bw..",1;"..item["label"]..";"..esc(item["question"]).."]"
+		pos = pos + 1
+	end
+	formspec = formspec.."button[0.25,"..pos..";"..bw..",1;new_question;+]"
+	pos = pos + 1
+	return formspec, pos
+end
+
+---handle main form fields
+---@param self table mob object
+---@param clicker string playername
```

Comment:
```suggestion
---@param clicker Player player object
```
глянь, там похоже именно Player нужен, а не его имя